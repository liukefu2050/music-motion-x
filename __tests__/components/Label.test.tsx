import { render, screen } from "@testing-library/react"
import Label from "../../src/components/Label"

it("worked", () => {
  const { asFragment, rerender } = render(
    <Label text="test" style={{ color: "red" }} />,
  )

  expect(screen.getByText("#test")).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot()

  rerender(<Label text="test" />)
  expect(asFragment()).toMatchSnapshot()
})
