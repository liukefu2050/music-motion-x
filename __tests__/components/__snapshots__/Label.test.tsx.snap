// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`worked 1`] = `
<DocumentFragment>
  <span
    class=" absolute py-[6px] px-3 rounded-[200px] bg-secondary text-[12px] text-black bottom-[14px] right-[10px] font-bold leading-none"
    style="color: red;"
  >
    #test
  </span>
</DocumentFragment>
`;

exports[`worked 2`] = `
<DocumentFragment>
  <span
    class=" absolute py-[6px] px-3 rounded-[200px] bg-secondary text-[12px] text-black bottom-[14px] right-[10px] font-bold leading-none"
    style=""
  >
    #test
  </span>
</DocumentFragment>
`;
